### PR TITLE
Improve Admin Ascend viewing

### PR DESCRIPTION
**Pull request by sethherr**
I know this will cause some merge conflicts with the style update, but  ¯\\_(ツ)_/¯

I needed some extra tools for looking at Ascend imports.

Updated view:

Screen Shot 2019-04-23 at 4 17 03 PM

I also figured we needed a standardized admin subheader, so I've added that.

### CODE REVIEW

**Comment by sethherr** [comment]
... and we need to be able to view ascend bikes, and lightspeed bikes. Adding some new stuff to the `admin/bikes` header so that we can sort them (making use of the new `admin-subnav` class :wink:)

Screen Shot 2019-04-23 at 5 11 30 PM

Fixes #548 (when complete)

**Comment by sethherr** [comment]
After shipping, run this in console:

```ruby
CreationState.find_each { |c| c.update_attributes(updated_at: Time.now) }
```

File: app/models/bike.rb [hunk 1]
```diff
@@ -149,6 +150,16 @@ def get_listing_order
     stock_photo_url.present? || public_images.present? ? t : t/100
   end
 
+  def creation_state; creation_states.first end
+
+  def creation_description; creation_state&.creation_description end
+  
+  def bulk_import; creation_state&.bulk_import end
+
+  def pos_kind; creation_state&.pos_kind end
```
Comment (jmromer) [inline app/models/bike.rb]: Suggestion / Question — [delegate](https://apidock.com/rails/Module/delegate) provides a succinct way to define this behavior / Should we prefer the longhand form?

Comment (sethherr) [reply app/models/bike.rb]: Yeah. This changed it from being `delegate` - I couldn't get the join working when it was `has_one`, and I couldn't get delegate working without a relation.

There is probably a way to fix either of those, and I'd be happy to use it! I'd prefer delegate for sure. I just didn't want to mess with it too much.

Comment (jmromer) [reply app/models/bike.rb]: Ah gotcha. Just checking, thanks 👍

**Pull request merged**